frontends/web/src/style/variables.css

```diff
@@ -25,6 +25,7 @@
     --color-primary: var(--color-blue);
     --color-secondary: var(--color-gray);
     --color-light: var(--color-mediumgray);
```

Comment:
Searched, but `--color-light` isn't used anywhere, but `--color-mediumgray` is used in many places, remove `--color-light`  here or change all places to use `--color-light` instead of  `--color-mediumgray` ?

Reply:
Also `--color-gray` and `--color-gray-alt` is used in various places, so for now I would just remove `--color-light` here. And then look at mediumgray vs. gray vs. gray-alt later in a different PR.

Reply:
I searched `--color-mediumgray` and found that it is mostly used for backgrounds (and boarders). I found one instance where it is used as a font color and changed that to `--color-light`.

Since background and fonts shouldn't use the same variable, we need to make a new background variable to use `--color-mediumgray`.

Reply:
I also see we use `--color-gray-alt` which doesn't have a corresponding font color variable. So we either need to make one or change it to an existing font color variable such as `--color-secondary`.